packages/jaeger-ui/src/components/TracePage/TraceStatistics/tableValuesTestTrace/traceWithSingleChildSpanLongerThanParent.json

```diff
@@ -32,8 +32,8 @@
           "spanID": "006c3cf93508f205"
         }
       ],
-      "startTime": 1679437737491529,
-      "duration": 79182,
+      "startTime": 200,
+      "duration": 300,
```

Comment:
file name doesn't quite match - the child is not only longer, but starts way outside of parent range.

Reply:
`traceWithSingleChildLongerThanParentAndStartsAfterParent`

May be this would be good! But too long